src/main/java/peppa/Storage.java

```diff
@@ -0,0 +1,71 @@
+package peppa;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;

```

Comment:
I'm not sure if its better or part of coding standards but iirc in 2030S we used to leave an empty line for imports of different directories. I think it might be a good idea you could consider.

```suggestion
import java.io.FileWriter;
import java.io.IOException;

import java.util.Scanner;

```